src/main/java/no/nav/veilarboppfolging/service/AuthService.java

```diff
@@ -90,7 +97,20 @@ public void skalVereInternEllerSystemBruker() {
     }
 
     public boolean harEksternBrukerTilgang(Fnr fnr) {
-        return getInnloggetBrukerIdent().equals(fnr.get());
+        // Når man ikke bruker Pep så må man gjøre auditlogging selv
+        var subjectUser = getInnloggetBrukerIdent();
+        var isAllowed = subjectUser.equals(fnr.get());
+        auditLogger.log(CefMessage.builder()
```

Comment:
har man ikke behov for å følge med på om det er allowed eller denied i audit meldingen?